src/custom/simplifions/components/SimplifionsRecoSolutions.vue

```diff
@@ -1,48 +1,96 @@
 <template>
   <div class="reco-solution fr-p-2w fr-mb-2w">
     <h4 class="fr-h4 fr-mb-2w">
-      {{ reco_solution.Nom_de_la_solution_publique }}
+      ➡️ {{ reco_solution.Nom_de_la_solution_publique }}
     </h4>
 
-    <p>
-      <strong>
-        <span
-          aria-hidden="true"
-          style="color: #27a658"
-          class="fr-icon-success-fill"
-        ></span>
-        En quoi cette solution est utile pour ce cas d'usage ?
-      </strong>
-      <br />
-      {{
-        reco_solution.En_quoi_cette_solution_est_elle_utile_pour_ce_cas_d_usage_
-      }}
-    </p>
-
-    <p>
-      <strong>🧑 Concrètement, pour les usagers :</strong>
-      <br />
-      {{ reco_solution.Concretement_pour_les_usagers_ }}
-    </p>
-
-    <p>
-      <strong>🧑‍💼 Concrètement, pour vos agents :</strong>
-      <br />
-      {{ reco_solution.Concretement_pour_vos_agents_ }}
-    </p>
-
-    <p>
-      <strong>
-        <span
-          aria-hidden="true"
-          style="color: #ff292f"
-          class="fr-icon-error-fill"
-        ></span>
-        Ce que ne fait pas cette solution :
-      </strong>
-      <br />
-      {{ reco_solution.Ce_que_ne_fait_pas_cette_solution_ }}
-    </p>
+    <div class="fr-grid-row fr-grid-row--gutters fr-m-4w fr-grid-row--top">
+      <div class="fr-col-6 fr-col-md-3">
+        <div class="fr-tile fr-tile--sm fr-tile--vertical fr-enlarge-link">
+          <div class="fr-tile__body">
+            <div class="fr-tile__content">
+              <p class="fr-tile__title fr-text--lead">
+                <a
+                  v-if="relatedSolution"
+                  :href="`/solutions/${relatedSolution.slug}`"
+                >
+                  {{ reco_solution.Nom_de_la_solution_publique }}
+                </a>
+                <a v-else :href="reco_solution.url_solution">
+                  {{ reco_solution.Nom_de_la_solution_publique }}
+                </a>
+                <br />
+                <span class="fr-text-xs">En savoir plus</span>
```

Comment:
```suggestion
                <span class="fr-text--xs">En savoir plus</span>
```

Reply:
Ah oui mais alors c'est ridiculement petit après XD

Je vais enlever la classe qui ne sert à rien du coup, et je laisserai Dorine itérer sur l'UI.